Answers.md

```diff
@@ -1,5 +1,38 @@
 1.  Explain the differences between `client-side routing` and `server-side routing`.
+
+Server-Side Routing (Standard)
+
+Needs to refresh with every new request. The information requested is the will be the only information given.  This can make things slow if a lot of information Is being requested. 
+
+Client-Side Routing 
+
+Routing handled internally by the JS that is already on the page.  This is the modern way of routing. No refreshing at all and will display data on the drop of a dime because it is already stored. 
+
+Main difference is that server-side requires refresh/reload and Client side doesn't. 
+
```

Comment:
These are good answers. Something to add would be that the initial load time for Client-Side Routing is longer. Additionally another way to phrase what you've written is that Client-Side Routing pulls all the information that it needs for all routes at the beginning, and then only pulls new information if needed.